Change_Management_Policy.md

```diff
@@ -0,0 +1,71 @@
+contemplating or initiaing change we should always apply due diligence in the consideration of client needs while balancing
+our needs as well.
+
+### Background Statement 
+
+We, at Fonticons, know our culture but it is entirely reasonable for others planning to use our technology to desire to 
+understand how and why are software and services might change. This is especially true as some of our changes may not be 
+backward compatible or may have unforseen impacts (since our software is opensource and thus can and is often leveraged in 
+ways that we do not expect).
+
+## Definitions 
+
+### Terms
+
+* The word we shall mean Fonticons Inc, all Fonticons employees and any individuals contracting with Fonticons to complete work.
+* Employee shall mean an individual directly employeed by Fonticons Inc and all contractors, consultants, temporary employees, or business partners.
+* User shall mean any individual who is not an employee.
+* Client shall mean a person or entity who installs or configures part or all of the Font Awesome free product/service for use on a website or product not owned or otherwise controlled by Fonticons.
+* Pro client shall mean a person or entity who installs or configures part or all of the Font Awesome paid product/service for use on a website or product not owned or otherwise controlled by Fonticons.
+* Fonticons system shall mean any computers, communication systems, platforms, and any other information technology systems used by Fonticons Inc., to provide the Font Awesome product and service.
+* Font Awesome product shall mean the icons, their digital representation, and associated icon functionality present in code.
+* Font Awesome service shall mean the technologies that make the Font Awesome product available to clients, users, and site users.
+
+## Policy 
+
+1. This policy applies to changes that directly impact clients. Like for like functional changes such as those associated with operations or security follow a different policy. 
+1. All employees may suggest, request, or initiate a change to the Font Awesome product or service.
+1. Any user with access to GitHub may suggest or request a change to the Font Awesome product or service.
+1. All changes must either be classified as technical or design changes.
+   1. Technical changes are changes to the Font Awesome service or the aspects of the Font Awesome product that are not direct visual representations of icons and their metadata.
+   1. Design changes are changes to direct visual representations of icons and their metadata.
+1. Technical changes must be approved by the head of development.
+1. Design changes must be approved by the head of design.
+1. The head of development and head of design should take special care to consider the impact to clients and users within the bounds of expected usage.
+   1. To facilitate this policy Fonticons must maintain an up-to-date expected usage statement.
```

Comment:
So, if you pulled down a random SVG and used the data in a path to seed a random number generator, then we changed that path and it broke your RNG, that is your problem because that is NOT expected usage.